tests/arrows/jsfmt.spec.js

```diff
@@ -1,2 +1,2 @@
-run_spec(__dirname, ["babylon", "typescript"], { arrowParens: "avoid" });
-run_spec(__dirname, ["babylon", "typescript"], { arrowParens: "always" });
+run_spec(__dirname, ["babel", "typescript"], { arrowParens: "avoid" });
```

Comment:
(Unrelated to this file)

There's a missing case in https://github.com/wuweiweiwu/prettier/blob/7b7606de469e89773f298e729b000f6653f37639/tests/multiparser_js_html/jsfmt.spec.js#L1.

Reply:
Will add!